tony-history-server/app/controllers/JobsMetadataPageController.java

```diff
@@ -38,13 +39,14 @@ public Result index() {
     }
 
     List<JobMetadata> listOfMetadata = new ArrayList<>();
-    String tonyHistoryFolder = config.getString("tony.historyFolder");
+    Path tonyHistoryFolder = new Path(config.getString("tony.historyFolder"));
+    String jobFolderRegex = "^application_\\d+_\\d+$";
```

Comment:
Please define this as a `static final String` class variable. The naming convention for such variables is generally `UPPERCASE_UNDERSCORE_SEPARATED`.

Reply:
Fixed: https://github.com/linkedin/TonY/pull/81/commits/108fe8650f76b5d9d5ae71188f3b95d170b54a93.